src/index.html

```diff
@@ -1,12 +1,45 @@
-<!DOCTYPE html>
+  <!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="body">
+    <div class="calendar calendar--month-length_31">
+      <div class="calendar__day calendar__day--start-day_sun"></div>
```

Comment:
start-day should be a modifier for the calendar block
```suggestion
    <div class="calendar calendar--month-length_31 calendar--start-day_sun">
      <div class="calendar__day"></div>
```